brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -113,6 +114,24 @@ public boolean accepts(String planType) {
         }
     }
 
+
+    public EntitySpec<? extends Application> createApplicationSpec(Path plan) throws PlanNotRecognizedException {
+        assertAvailable();
+        try {
+            Alien4CloudToscaPlatform.grantAdminAuth();
+            return createApplicationSpec(platform.parse(plan));
+        } catch (Exception e) {
+            if (e instanceof PlanNotRecognizedException) {
+                if (log.isTraceEnabled())
```

Comment:
I copied the existing method.  What is the point of checking for the `PlanNotRecognizedException` only to do the same thing? Do you know what the intention behind the original is?